scripts/publish_vessel_info

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+source pipe-tools-utils
+
+THIS_SCRIPT_DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
+ASSETS=${THIS_SCRIPT_DIR}/../assets
+ARGS=( \
+  SOURCE \
+  TEMP_BUCKET \
+  ELASTIC_SEARCH_SERVER_URL \
+  ELASTIC_SEARCH_SERVER_AUTH \
+  ELASTIC_SEARCH_INDEX_ALIAS \
+)
+
+################################################################################
+# Validate and extract arguments
+################################################################################
+display_usage() {
+  echo -e "\nUsage:\n$0 SOURCE TEMP_BUCKET ELASTIC_SEARCH_SERVER_URL ELASTIC_SEARCH_SERVER_AUTH ELASTIC_SEARCH_INDEX_ALIAS\n"
+}
+
+if [[ $# -ne ${#ARGS[@]} ]]
+then
+    display_usage
+    exit 1
+fi
+
+ARG_VALUES=("$@")
+PARAMS=()
+for index in ${!ARGS[*]}; do
+  echo "${ARGS[$index]}=${ARG_VALUES[$index]}"
+  declare "${ARGS[$index]}"="${ARG_VALUES[$index]}"
+done
+
+
+TEMP_PATH=gs://${TEMP_BUCKET}/pipe-vessels/$( date -u "+%FT%T.%N" )
+
+echo "Running $0"
+echo "  SOURCE: $SOURCE"
+echo "  TEMP_BUCKET: $TEMP_BUCKET"
+echo "  ELASTIC_SEARCH_SERVER_URL: $ELASTIC_SEARCH_SERVER_URL"
+echo "  ELASTIC_SEARCH_SERVER_AUTH: [HIDDEN]"
+echo "  ELASTIC_SEARCH_INDEX_ALIAS: $ELASTIC_SEARCH_INDEX_ALIAS"
+################################################################################
+# Export records to json files
+################################################################################
+echo "Exporting records from $SOURCE"
+EXTRACT_PATH=$TEMP_PATH/bq/*.json
```

Comment:
@andres-arana I think I preffer to have a `bq query` here instead of an extract, that will let us control a bit more what ends up being on the files. 

Also I think that it should be a pre-requisite that Elastcisearch receives the data on the needed format `YYYY-MM-DDT00:00:00Z`: https://github.com/GlobalFishingWatch/pipe-vessels/pull/1/files#diff-5973c79e28ebe22fefe322f6a76a9b04R18 and that should be achieved here in the `bq query`